android/build/mfx_defs_internal.mk

```diff
@@ -0,0 +1,34 @@
+# Purpose:
+#   Defines include paths, compilation flags, etc. to build Media SDK
+# internal targets (libraries, test applications, etc.).
+#
+# Defined variables:
+#   MFX_CFLAGS_INTERNAL - all flags needed to build MFX targets
+#   MFX_CFLAGS_INTERNAL_HW - all flags needed to build MFX HW targets
+#   MFX_C_INCLUDES_INTERNAL - all include paths needed to build MFX targets
+#   MFX_C_INCLUDES_INTERNAL_HW - all include paths needed to build MFX HW targets
+
+MFX_CFLAGS_INTERNAL := \
+    $(MFX_CFLAGS) \
+    $(MFX_CFLAGS_STL)
+
+MFX_CFLAGS_INTERNAL_32 := -DLINUX32
+MFX_CFLAGS_INTERNAL_64 := -DLINUX32 -DLINUX64
+
+MFX_CFLAGS_INTERNAL_HW := \
+    $(MFX_CFLAGS_INTERNAL) \
+    -Wno-error=non-virtual-dtor
```

Comment:
can you check whether we really still need this?

Reply:
It was added by @dmitryermilov with the following message: "Fix for "shared library text segment is not shareable" warrning treated as error."
I checked the build without this line, and the build was completed without such error.

Reply:
done